packages/app/src/sections/futures/MobileTrade/UserTabs/TransfersTab.tsx

```diff
@@ -57,40 +57,41 @@ const TransfersTab: React.FC = () => {
 					highlightRowsOnHover
 					columns={[
 						{
-							Header: <TableHeader>{t('futures.market.user.transfers.table.action')}</TableHeader>,
-							accessor: 'action',
-							Cell: (cellProps: any) => <ActionCell>{cellProps.value}</ActionCell>,
-							width: 50,
+							header: () => (
+								<TableHeader>{t('futures.market.user.transfers.table.action')}</TableHeader>
+							),
+							accessorKey: 'action',
+							cell: (cellProps: any) => <ActionCell>{cellProps.value}</ActionCell>,
+							size: 50,
 						},
 						{
-							Header: <TableHeader>{t('futures.market.user.transfers.table.amount')}</TableHeader>,
-							accessor: 'size',
-							sortType: 'basic',
-							Cell: (cellProps: any) => {
-								const formatOptions = {
-									minDecimals: 0,
-								}
+							header: () => (
+								<TableHeader>{t('futures.market.user.transfers.table.amount')}</TableHeader>
+							),
+							accessorKey: 'size',
+							sortingFn: 'basic',
+							cell: (cellProps) => {
+								const formatOptions = { minDecimals: 0 }
```

Comment:
Wonder if this is actually doing anything?

Reply:
Removed. Also this tab isn't actually being used lol